src/custom-elements/src/components/date/date.tsx

```diff
@@ -232,14 +267,23 @@ export class Date {
     this.updateExcludedDates();
     this.updateExcludedWeekdays();
     this._initCalendarViewModel();
-    this.viewMonth = this.value ? this.fromString(this.value) : dayjs();
+    this._initViewMonth();
     this._updateViewMonth();
   }
 
+  deselectDate(day: Dayjs) {
+    const formattedDate = this.toDayString(day);
+    const currentValues = Array.from(this.value.split(','));
+
+    this.value = currentValues.filter(value => value !== formattedDate).join(',');
+    this.eventChange.emit(this.value);
+  }
+
```

Comment:
If you create a method `addDate` this should be named `removeDate`. Makes sentido?

Reply:
Totally makes sentido. Will make the changes. Thanks Dani!